### DISCUSSION TITLE
win11下打开eso会报错，无法使用RangeError ：Not enough byes available

### DISCUSSION THREAD

**Discussion by bauw2008** [mabDc/eso#68] [General]
版本：ESO-1.23.5-12305 eso.windows.release.zip
具体错误 如下图：

![屏幕截图 2023-05-10 212706](https://github.com/mabDc/eso/assets/77882800/8a642d44-6acc-4ef0-9236-12dd9fac461d)

**Comment by mabDc** [comment]
会不会是旧版升级需要清理数据

**Reply by bauw2008** [reply]
感谢回复，旧版本好像还是ESO-1.21.05-12105没重写前的好像，我过会试试。

**Reply by mabDc** [reply]
> 感谢回复，旧版本好像还是ESO-1.21.05-12105没重写前的好像，我过会试试。

我很抱歉让你遇到bug，最近没有上传github(原来这里这么回复，展开像贴吧。

**Reply by bauw2008** [reply]
这个问题解决了，删除C:\Users\xxx\OneDrive\資料\eso这个目录内容
